app/views/api/v2/layouts/permissions.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @resource
+
+if params.has_key?(:include_permissions)
+  @authorizer ||= authorizer
```

Comment:
don't you already get this caching from the AuthorizeHelper's `authorizer` method?